trusted-committer/es/07-becoming-a-trusted-committer-es.asciidoc

```diff
@@ -0,0 +1,44 @@
+== Convirtiendose en un Trusted Committer
+
+
+El rol del Trusted Committer es un rol demandante pero gratificante.
+Si este camino de aprendizaje te interesa, te puedes estar preguntando como te puedes convertir en un Trusted Committer y si eres la persona indicada para el trabajo.
+
+Las comunidades de InnerSource siguen los mismos principios que las comunidades Open Source, uno de estos es la meritocracia.
+En una meritocracia, el poder se da a los individuos basados en su talento, esfuerzo y logros.
+Esto significa que el poder y los privilegios que vienen de ser un Trusted Committer deben de ganarse.
```

Comment:
Please remove the "**_de_**" between "_deben_" and "_ganarse_". "_deben_" alone expresses an obligation whereas "_deben de_" means a guess or an assumption.